algebras/algebra/src/main/scala/endpoints/algebra/Urls.scala

```diff
@@ -97,6 +97,9 @@ trait Urls {
   /** An URL path carrying an `A` information */
   type Path[A] <: Url[A]
 
+  /** Implicit conversion to get rid of intellij errors when defining paths. Effectively should not be called.*/
+  implicit def dummyPathToUrl[A](p: Path[A]): Url[A] = p
```

Comment:
So I kind of agree in principle, but I've seen at least 3 times people thinking they have something wrong when being introduced to endpoints because intellij marks it as red. It doesnt make the experience better.

Reply:
OK then!